index.Rmd

```diff
@@ -1,26 +1,29 @@
 ---
-title: "Course Name"
+title: "Informatics Technology for Cancer Research (ITCR): Cancer Informatics Data Visualization in Practice"
 date: "`r format(Sys.time(), '%B, %Y')`"
 site: bookdown::bookdown_site
 documentclass: book
-bibliography: [book.bib]
+bibliography: [book.bib, packages.bib]
 biblio-style: apalike
 link-citations: yes
-description: "Description about Course/Book."
-favicon: assets/dasl_favicon.ico
-output:
-    bookdown::word_document2:
-      toc: true
+description: "This ITN course focuses on practicing best practices in data visualization, specifically for cancer informatics data and research questions, by discussing common graph types used within Cancer Informatics, building graphs (while employing best practices), and showcasing ITCR tools that are helpful for data visualization of cancer research data."
+favicon: assets/ITN_favicon.ico
 ---
 
-# About this Course {-}
 
+```{r include=FALSE}
+# automatically create a bib database for R packages
+knitr::write_bib(c(
+  .packages(), "bookdown", "knitr", "rmarkdown"
+), "packages.bib")
+```
 
-## Available course formats
+# About this Course {-}
 
-This course is available in multiple formats which allows you to take it in the way that best suites your needs. You can take it for certificate which can be for free or fee.
+This course is part of a series of courses for the [Informatics Technology for Cancer Research (ITCR)](https://www.cancer.gov/about-nci/organization/cssi/research/itcr). This material was created by the ITCR Training Network (ITN)  which is a collaborative effort of researchers around the United States to support cancer informatics and data science training through resources, technology, and events. This initiative is funded by the following grant:  [National Cancer Institute (NCI)](https://www.cancer.gov/)  UE5 CA254170. Our courses feature tools developed by ITCR Investigators and make it easier for principal investigators, scientists, and analysts to integrate cancer informatics into their workflows. Please see our website at [www.itcrtraining.org](www.itcrtraining.org) for more information.
 
-- The material for this course can be viewed without login requirement on this [Bookdown website](LINK HERE). This format might be most appropriate for you if you rely on screen-reader technology.
-- This course can be taken for [free certification through Leanpub](LINK HERE).
-- This course can be taken on [Coursera for certification here](LINK HERE) (but it is not available for free on Coursera).
-- Our courses are open source, you can find the [source material for this course on GitHub](LINK HERE).
+Cancer Informatics Data Visualization in Practice is the second in a series of two courses on data visualization. This course focuses on using the best practice considerations discussed in the first course, specifically within context of Cancer Informatics Research. This course accomplishes this by discussing topics such as
```

Comment:
could add major objectives to the other course here too - I was actually a tad confused which course you were referring to first so that might help fix that for people reading quickly like me :P

Reply:
Sorry, could you clarify what I should add here?